perf/terraform/modules/short_lived/main.tf

```diff
@@ -19,7 +19,7 @@ resource "aws_instance" "perf" {
 
   launch_template {
     name = "perf-node"
-    version = "3"
+    version = "1"
```

Comment:
Same here. In the libp2p AWS account the launch template to use is `version` `3`. Mind reverting this change? Otherwise I can not trigger CI.